optuna/samplers/_brute_force.py

```diff
@@ -25,16 +25,14 @@ class BruteForceSampler(BaseSampler):
 
     Note:
         The defined search space must be finite. Therefore, when using
-        :class:`~optuna.distributions.FloatDistibution`, `step=None` is not allowed.
+        :class:`~optuna.distributions.FloatDistibution`, ``step=None`` is not allowed.
 
     Note:
-        This sampler should not be used in combination with other samplers. For example,
-        in distributed optimization, if :class:`~optuna.samplers.BruteForceSampler` and another
-        sampler run optimizations simultaneously, complete exhaustive search may not be performed.
+        This sampler assumes that it suggests all parameters and that the search space is fixed.
```

Comment:
I felt this sentence was a bit ambiguous. Users might read it as `all values of parameters without context. How about specifying `combinations` explicitly?


```suggestion
        This sampler assumes that it suggests all combinations of parameters and that the search space is fixed.
```

Reply:
Here, I want to say that any parameters are not suggested by other features (other samplers / `enqueue_trial`).

Reply:
Ah, I misunderstood. Thank you for your explanation.